llumnix/backends/vllm/llm_engine.py

```diff
@@ -238,7 +239,13 @@ def __init__(
 
     def _start_engine_loop(self) -> None:
         while True:
-            self.engine.step()
+            try:
+                self.engine.step()
+            # pylint: disable=broad-except
+            except Exception as e:
+                logger.error("Error in engine loop: {}".format(e))
```

Comment:
1. reference to manager, following the same output format and print the traceback. 2. why exit?

Reply:
to release the resources

Reply:
Can we directly raise to achieve the same effect?

Reply:
raise the error can't exit the main thread. It will exit engine.step thead only.